src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index_fr.properties

```diff
@@ -0,0 +1,2 @@
+title=Titre de la page
+description=Les barres d'applications fournissent l'en-tête de la page ainsi que des actions importantes à propos de la page actuelle. Elles doivent être incluses avant le <code>l\:main-panel</code>.
```

Comment:
The jelly code looks like it was incorrectly escaped.  Should be `l:main-panel` rather than `l\:main-panel`.

Reply:
I'll take a look at it later 👀